components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -2122,8 +2123,8 @@ private OMElement queryToGetAPIUsageByDestination(String columnFamily, String fr
         }
     }
 
-    private OMElement queryBetweenTwoDaysForAPIUsageByVersion(String columnFamily, String fromDate, String toDate,
-                                                              String apiName)
+    private List<APIUsage> queryBetweenTwoDaysForAPIUsageByVersion(String tableName, String fromDate, String toDate,
+            String apiName)
```

Comment:
rename method as "getUsageByAPIVersionsData" and fixed in commit: https://github.com/wso2/carbon-apimgt/commit/006d2e055000123aa5f1e15f2b21a85b8003b621